README.md

```diff
@@ -1,8 +1,68 @@
-# Boas vindas ao primeiro projeto A CLI of Ice and Fire!
+# Boas vindas ao repositório do projeto de A CLI of Ice and Fire!
```

Comment:
O README já está sendo ajustado nesse [PR](https://github.com/betrybe/sd-0x-a-cli-of-ice-and-fire/pull/2/files), era de fato para o README ser alterado neste PR? 🤔 

Esses arquivos de gif eram também para estar neste PR?

Reply:
nao eram pra estar alias nen commitei eles

Reply:
Você poderia remover então tanto o `README` quanto os arquivos de gif nesta branch sua